### PR TITLE
Enable functional tests in CI

### PR DESCRIPTION
**Pull request by johnthagen**
# Testing

## Description

Enable functional testing in CI

Subtask of #369

### CODE REVIEW

**Comment by johnthagen** [comment]
@Archmonger Could you please review this PR? This only runs functional tests on SQLite, as far as I can tell, but it's at least better than not running them at all.

File: tox.ini [hunk 1]
```diff
@@ -16,11 +16,11 @@ commands = {posargs:coverage run runtests.py}
 # Configure which test environments are run for each Github Actions Python version.
 [gh-actions]
 python =
-    3.6: py36-django{22,32}
-    3.7: py37-django{22,32}
-    3.8: py38-django{22,32,40}
-    3.9: py39-django{22,32,40}
-    3.10: py310-django{22,32,40}
+    3.6: py36-django{22,32},functional
```
Comment (Archmonger) [inline tox.ini:19]: I'm not familiar with the function tox keyword. Quick google search didn't yield results.

Can you explain it's purpose?

Comment (Archmonger) [reply tox.ini:19]: Oh scratch that, its the test environment called functional.

**Review by Archmonger — APPROVED** [review]

**Comment by Archmonger** [comment]
If you click "join" within the [django-dbbackup jazzband group](https://jazzband.co/projects/django-dbbackup), you should be able to merge.

**Pull request merged**